crypto/ec/ec_lib.c

```diff
@@ -1025,3 +1025,11 @@ int EC_GROUP_do_inverse_ord(const EC_GROUP *group, BIGNUM *res,
     else
         return 0;
 }
+
+int ec_point_blind_coordinates(const EC_GROUP *group, EC_POINT *p, BN_CTX *ctx)
+{
+    if (!group->meth->blind_coordinates)
+        return -2; /* not implemented */
```

Comment:
Why -2 here? Is there a copy&paste from somewhere else? Do we actually need to distinguish the "not implemented" state - or perhaps it is enough that non-implemented blinding always "succeeds".

Reply:
@mattcaswell I took the `-2 == not_implemented` as a general pattern in EVP_PKEY wrappers.

I think that distinguishing the not_implemented case can be beneficial, if in the future higher level functions REQUIRE coordinate blinding to guarantee safety.

Do you think it's overkill to plan for such case now?

Reply:
> @mattcaswell I took the -2 == not_implemented as a general pattern in EVP_PKEY wrappers.

I dislike the "magic number". If we have more cases than 1 for success/0 for failure I think we should, in general, introduce some named values (e.g. via an enum, or #define). Although that's not something we've been very good at elsewhere so I wouldn't insist on it.

If we have to have a third return value other than 0 or 1, then I don't feel we have to follow what EVP_PKEY does. Elsewhere we have just used 1 == success, 0 == failure, -1 == error.

> Do you think it's overkill to plan for such case now?

I'm not sure. But functions in EC_METHODs are not publicly accessible so we're not exposing these return values in the public API in any way. That means its a lot easier to change later if needs be. I'd be tempted to not worry about that case now, and add it later if we need to.

Reply:
The issues are addressed ! We have changed the return value to 1 in case the underlying
EC_METHOD does not support coordinate blinding, and we also rebased on master.
There is also some documentation for the wrapper explaining the return values.